src/webview/wkwebview/mod.rs

```diff
@@ -907,6 +907,15 @@ r#"Object.defineProperty(window, 'ipc', {
     self.navigate_to_url(url, Some(headers))
   }
 
+  pub fn clear_all_browsing_data(&self) -> Result<()> {
+    unsafe {
+      let store: id = msg_send![class!(WKWebsiteDataStore), defaultDataStore];
+      let all_data_types: id = msg_send![class!(WKWebsiteDataStore), allWebsiteDataTypes];
+      let _: () = msg_send![store, removeDataOfTypes:all_data_types modifiedSince:null::<*const c_void>() completionHandler:null::<*const c_void>()];
```

Comment:
I am not so sure about this implementation at all and someone with macOS needs to test it

Reply:
I tested it by setting `document.cookie` manually and call this method.
But I agree more people to test is always better.

Reply:
I haven't seen that you fixed the implementation in [36f3d5f](https://github.com/tauri-apps/wry/pull/915/commits/36f3d5f9310eab78117a7afe61365133a736ea5b), I was just concerned with my own implementation. This should be fine to merge now I guess